arch/xtensa/src/esp32/esp32_serial.c

```diff
@@ -639,16 +639,21 @@ static void esp32_dmasend(struct uart_dev_s *dev)
     {
       struct esp32_dmadesc_s *dmadesc;
       uint8_t *tp;
-    #ifdef CONFIG_XTENSA_IMEM_USE_SEPARATE_HEAP
+    #ifdef CONFIG_ESP32_SPIRAM
       uint8_t *alloctp = NULL;
     #endif
 
-      /* If the buffer comes from PSRAM, allocate a new one from DRAM */
+      /* If the buffer comes from PSRAM, allocate a new one from SRAM */
```

Comment:
DRAM here refers to part of internal SRAM dedicated to Data (similarly as IRAM refers to the Instruction equivalent)

If you believe DRAM might be confusing, I suggest to instead mention **Internal SRAM**.